changelogs/CHANGELOG-v2.6.rst

```diff
@@ -15,15 +15,15 @@ Release Summary
 Minor Changes
 -------------
 
-- Sceanrio guide for removing an existing virtual machine is added.
```

Comment:
This file is generated and should not be edited.

It will be regenerated from the changelog fragments during the next release.